todo.js

```diff
@@ -15,3 +15,54 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+let process = require('process');
+let command = process.argv[2];
+let secondCommand = process.argv[3];
+// takes the third argument because the first two are the node and the location of the file
+let fs = require('fs');
+
+// file system, reads files
+
+function openAndRead(filename) {
+  let text = fs.readFileSync(filename, 'utf-8');
+  text = text.split('\n');
+  console.log('Here is your list!');
+  for (let i = 0; i < text.length - 1; i += 1) {
+    console.log(` ${i + 1}. ${text[i]} `);
+    // prints out the literal string, you put the data within the ${} when it is a variable and not a string
+  }
+}
+
+function writingList() { // no parameter because command is on the command line
+  if (command === 'list') {
```

Comment:
1. Avoid global variables. Where does `command` come from? You can always pass it in as an argument.
2. As you've written it, *every* function has to care about the value of `command`. What if you want to change the name of the variable? Or change what its value represents? You'd have to change every function in your code.

   Try to isolate the part of your program that is concerned with handling command-line arguments from everything else. If you want to change how command-line arguments are handled, ideally there's only one part of the code you have to edit.